dashboard/app/views/levels/new.html.haml

```diff
@@ -14,6 +14,7 @@
 %h3 Content
 %ul
   %li= link_to 'Build a Standalone Video Level', new_level_path(type: 'StandaloneVideo')
+  %li= link_to 'Build an External Link Level', new_level_path(type: 'ExternalLink')
```

Comment:
Maybe put this down below in the "CSP Apps" section — the `link_title` isn't localizable so these levels will be en_us only.